app/assets/javascripts/leaflet.query.js

```diff
@@ -8,9 +8,23 @@ L.OSM.query = function (options) {
     const link = $("<a>")
       .attr("class", "control-button")
       .attr("href", "#")
-      .html("<span class=\"icon query\"></span>")
       .appendTo($container);
 
+    $(L.SVG.create("svg"))
+      .append(
+        $(L.SVG.create("path")).attr("d", "M3 1s9 7 9 7c-1.056.587-2.112 1.173-3.168 1.76l3.031 6.615c.346.753.015 1.643-.738 1.989-.753.345-1.643.014-1.989-.739L6.201 11.221C5.134 11.814 4.067 12.407 3 13V1z"),
+        $(L.SVG.create("text")).text("?").attr({
+          "x": "12.5",
+          "y": "18.512",
+          "font-family": "Helvetica, sans-serif",
+          "font-weight": "Bold",
+          "font-size": "12"
+        })
+      )
+      .attr("fill", "currentColor")
+      .attr("class", "h-100 w-100")
+      .appendTo(link);
```

Comment:
Is this easier to read?
```suggestion
    $(`<svg>
    <path d="M3 1s9 7 9 7c-1.056.587-2.112 1.173-3.168 1.76l3.031 6.615c.346.753.015 1.643-.738 1.989-.753.345-1.643.014-1.989-.739L6.201 11.221C5.134 11.814 4.067 12.407 3 13V1z" />
    <text "x": "12.5" "y": "18.512" 
    "font-family"="Helvetica, sans-serif" "font-weight"="Bold" "font-size"="12"
    "class"="h-100 w-100"
    "fill"="currentColor" 
    >?</text></svg>`)
      .appendTo(link);
```
Probably easier to process by the browser...

Reply:
Probably best to define somewhere else collectively in the html and just reuse, to get the clip path out of the css too.
But where (if not in a sprite)?

Reply:
Oh, you are right. 
DOM APIs like `innerHTML` and `insertAdjacentHTML` creates valid svg elements. jQuery does not with this api.

Reply:
I'd put the svgs into `app/views/layouts/map.html.erb` but some non-map layout pages like *Dashboard* also use them. So I guess it should be a template that is included by the map layout and those other pages.

Reply:
Doesn't the dashboard use the map layout via the site controller?

Reply:
Dashboard has its own controller `app/controllers/dashboards_controller.rb`.